src/components/ToggleSwitch/index.tsx

```diff
@@ -63,7 +64,7 @@ const styles = StyleSheet.create({
   mainStyes: {
     position: 'relative',
     borderRadius: dimen.normalizeDimen(12),
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, this change replaces a hard-coded color value with a dynamic one from a theme object, improving modularity and consistency.
```diff
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->